From a5c2f12cd48a82cbc1f53de64cf81c37fcdfcdb8 Mon Sep 17 00:00:00 2001
From: Ariel Aviad <aaviad@habana.ai>
Date: Wed, 23 Apr 2025 16:52:45 +0300
Subject: [PATCH] [SW-225888] g3: serdes: remove usage of saved serdes partition file

Remove usage of saved SerDes partition file we are
no longer using it and creating it as part of the build

Change-Id: I036ce3dde586ab27bc3df113b81203fdd046ed8f
Signed-off-by: Ariel Aviad <aaviad@habana.ai>
---

diff --git a/board/habanalabs/gaudi3/serdes/FlashMemory.bin b/board/habanalabs/gaudi3/serdes/FlashMemory.bin
deleted file mode 100644
index 823e279..0000000
--- a/board/habanalabs/gaudi3/serdes/FlashMemory.bin
+++ /dev/null
Binary files differ
